feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/usecase/ObserveSyncInfoUseCase.kt

```diff
@@ -329,3 +329,17 @@ internal class ObserveSyncInfoUseCase @Inject constructor(
         private const val COUNT_EVENTS_CACHE_LIFESPAN_MILLIS = 10 * 1000L
     }
 }
+
+private sealed class EventSyncVisibleSection
```

Comment:
My comment about naming is actually more relevant here as the var name comes from the class name.